### ISSUE TITLE
Keyboard hides Search textfield

### ISSUE THREAD

**Issue by 92sajid110** [AbdullahChauhan/custom-dropdown#51]
Hello,
Hope you are good.
I am using your nice package in my projects but at one place i found an issue.
1.CustomDropdown<String>.search() is place at bottom of screen assume at 70% & 80% of screen
so when textfield is clicked keyboard hides search field and user can't see what is typing
The layout is like this that i can't move it to top as in top there is other dropdown and all linked with data validation.
So i added scaffold,singlechildscrollview but no effect.
Respected behaviour is scroll to top of  keyboard.

2. In same page there is an textfield and after that there is all dropdowns with search.
So basically user enter value in textfield and click on below dropdown, so i want a callback or any params to dismiss or perform any task whenever user taps on dropdown before opening dropdown this way i can dismiss keyboard or unfocus above keyboard.

3. If possible add params to autofocus(bool) on CustomDropdown<String>.search()to autofocus on textfield after opening dropdown

Thanks

**Comment by hbatalhaStch** [comment]
I forked the project and made changes in the dropdown overlay so that it always shows at the top when I wanted: https://github.com/hbatalhaStch/custom-dropdown/tree/project

I don't know if you still have this issue but it might be useful for new comers.

**Comment by KabaDH** [comment]
@hbatalhaStch hi! Would you mind share a link to commit with your workaround, please? I could not find it in your repo.

**Comment by hbatalhaStch** [comment]
> @hbatalhaStch hi! Would you be so kind as to share the link to the commit with your workaround, please? I couldn't find it in your repo.

here it is: https://github.com/hbatalhaStch/custom-dropdown/commit/8bbda70a9e0e615d7e7ec0ab2c571d87742d98cf

**Comment by jaswantk917** [comment]
hiding behind keyboard

**Comment by therjtkumar** [comment]
still hiding behind keyboard.

**Comment by FlawLessx** [comment]
i there any fix?

**Comment by zeelkumar-sojitra** [comment]
Still hiding behind keyboard.
